### PR TITLE
Deal with items properties in mocks

### PR DESCRIPTION
**Pull request by Henry-Pulver**
Previously:
```
{
"content": {
"properties": {
"items": {
"contactId": 123456,
"sessionId": 12345,
"score": 4,
"feedback": "Great session! Learned a lot.",
"rated": "2023-08-20T15:30:00Z"
}
}
}
}
```

Now:
```
{
"content": {
"contactId": 123456,
"sessionId": 12345,
"score": 4,
"feedback": "Great session! Learned a lot.",
"rated": "2023-08-20T15:30:00Z"
}
}
```

Note: there are still issues with mocking:
1. It mocks values which then are removed by post-processing
2. It includes `properties` and `items` in the output of the LLM unnecessarily

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -1,8 +0,0 @@
-{
-  "functions": {
-    "pages/api/receive-push.ts": {
-      "memory": 500,
-      "maxDuration": 60
-    }
-  }
-}
```
Comment (neuromantic99) [inline vercel.json:8]: :O very important file ;)

File: __tests__/apiMock/removePropertiesItems.test.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    expect(out).toEqual(expected);
+  });
+  it("nested in properties", () => {
+    const out = removePropertiesItems({ properties: expected });
+    expect(out).toEqual(expected);
+  });
+  it("nested in items", () => {
+    const out = removePropertiesItems({ properties: expected });
+    expect(out).toEqual(expected);
+  });
```
Comment (neuromantic99) [inline __tests__/apiMock/removePropertiesItems.test.ts:17]: these two tests be the same matey

File: __tests__/apiMock/removePropertiesItems.test.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  it("double nested with outer non-removed key", () => {
+    const expected = { out: 1, nowt: 2 };
+    const out = removePropertiesItems({
+      key: { items: expected, andAnother: "one" },
+      another: "one",
+    });
+    expect(out).toEqual({
+      key: { ...expected, andAnother: "one" },
+      another: "one",
+    });
```
Comment (neuromantic99) [inline __tests__/apiMock/removePropertiesItems.test.ts:53]: can you add a test for arrays pretty please

File: __tests__/apiMock/removePropertiesItems.test.ts (hunk 2 above)
Comment (Henry-Pulver) [reply __tests__/apiMock/removePropertiesItems.test.ts:17]: Oops!

File: __tests__/apiMock/removePropertiesItems.test.ts (hunk 3 above)
Comment (Henry-Pulver) [reply __tests__/apiMock/removePropertiesItems.test.ts:51]: Added 2

**Pull request merged**